app/src/main/java/org/proninyaroslav/libretorrent/core/utils/Utils.java

```diff
@@ -68,6 +68,7 @@
     public static final String MAGNET_PREFIX = "magnet";
     public static final String HTTP_PREFIX = "http";
     public static final String HTTPS_PREFIX = "https";
+    public static final String INFOHASH_PREFIX = "magnet:?xt=urn:btih:";
```

Comment:
Is there a reason to not include "btih:" on this string? I assume this app only uses the BitTorrent protocol but if it does not, we'd need to remove this and require the user to add that part of the prefix themselves / add a spinner for options to select.